src/base/settingsstorage.h

```diff
@@ -62,4 +62,27 @@ public slots:
     mutable QReadWriteLock m_lock;
 };
 
+template<typename T>
+class SettingsStorageHelper
```

Comment:
The next step is to rename the class into `StorableValue`, add `operator T` and `operator =(T)` and just read and write them lol